docs/using/components/atomic/current_public_table.rst

```diff
@@ -1,3 +1,7 @@
+********************************
+Current Public Atomic Data Table
```

Comment:
That's good thing to do.

BTW is it referred anywhere - it is apparently not listed in any toctree I think?

Reply:
It is, but it doesn't appear in the toctrees because it had no title.